client/src/PublicRequest.ts

```diff
@@ -299,6 +306,13 @@ export type SignSwapRequestCommon = SimpleRequest & {
             // bankLogoUrl?: string,
             // bankColor?: string,
         }
+    ) | (
+        {type: 'CRC'}
+        & {
+            amount: number,
+            fee: number,
+            recipientLabel?: string,
```

Comment:
Since this is the funding side, the name `recipientLabel` doesn't make sense. Below, you are parsing this into `sinpeLabel`. `senderLabel` also makes sense.